### ISSUE TITLE
108.0b1 not compatible

### ISSUE THREAD

**Issue by aprojanos71** [RealRaven2000/quickFilters#138]

**Comment by RealRaven2000** [comment]
test version below:

[quickFilters-wx-5.7pre28.zip](https://github.com/RealRaven2000/quickFilters/files/10096544/quickFilters-wx-5.7pre28.zip)

**Comment by aprojanos71** [comment]
Hi! 109.0b2 not compatible

**Comment by RealRaven2000** [comment]
New test version:

[quickFilters-wx-5.7.2pre1.zip](https://github.com/RealRaven2000/quickFilters/files/10319207/quickFilters-wx-5.7.2pre1.zip)

thanks for testing - if you find any beta-specific regressions (unrelated to [changes in v5.7.1](https://github.com/RealRaven2000/quickFilters/releases/tag/5.7.1)) please let me know below.

**Comment by RealRaven2000** [comment]
New version for Thunderbird 102 beta, below. I noticed the "Message Filters" button stopped working, so this version repairs that. Tb also currently has a bug that doesn't allow vertical resizing of the filters list.

[quickFilters-wx-5.8pre71.zip](https://github.com/RealRaven2000/quickFilters/files/10549008/quickFilters-wx-5.8pre71.zip)

To install, download archive and drop into Thunderbird Add-ns Manager (don't extract contents).

**Comment by RealRaven2000** [comment]
Latest version, can be installed on TB 124 betas.

[quickFilters-wx-6.3.1pre6.zip](https://github.com/RealRaven2000/quickFilters/files/14366557/quickFilters-wx-6.3.1pre6.zip)

To install, download archive and drop into Thunderbird Add-ns Manager (don't extract contents).

**Comment by RealRaven2000** [comment]
test version for 125 beta below:

[quickFilters-wx-6.4pre26.zip](https://github.com/RealRaven2000/quickFilters/files/14760857/quickFilters-wx-6.4pre26.zip)

-------

To install, download archive and drop into Thunderbird Add-ns Manager (don't extract contents).

**Comment by RealRaven2000** [comment]
test version for 126 beta below:

[quickFilters-wx-6.5pre5.zip](https://github.com/RealRaven2000/quickFilters/files/15119135/quickFilters-wx-6.5pre5.zip)

---

To install, download archive and drop into Thunderbird Add-ns Manager (don't extract contents).

**Comment by RealRaven2000** [comment]
Test version for 127 beta below:

[quickFilters-wx-6.5pre16.zip](https://github.com/RealRaven2000/quickFilters/files/15362786/quickFilters-wx-6.5pre16.zip)


-----

To install, download archive and drop into Thunderbird Add-ns Manager (don't extract contents).

**Comment by RealRaven2000** [comment]
Latest test version, compatible with beta 128 - which is close to the final release of Thunderbird ESR 128.0 planned for this summer.

[quickFilters-wx-6.5pre20.zip](https://github.com/user-attachments/files/15836872/quickFilters-wx-6.5pre20.zip)

------

To try the version above download zip file and drag the file into Thunderbird Add-ons Manager (do not extract contents, it won't install like that)

**Comment by RealRaven2000** [comment]
Latest version - compatible with Tb 128.*, tested on current beta 128.0b3. This also fixes #244

[quickFilters-wx-6.5pre21.zip](https://github.com/user-attachments/files/15968917/quickFilters-wx-6.5pre21.zip)

----

To try the version above download zip file and drag the file into Thunderbird Add-ons Manager (do not extract contents, it won't install like that)

**Comment by RealRaven2000** [comment]
Compatibility with ESR 128 Implemented 
v6.5.1 - published 16/07/2024

**Comment by BillH99999** [comment]
Where do we find version 6.5.1?  https://addons.thunderbird.net/en-US/thunderbird/addon/filtaquilla/versions/ only shows 4.0.

**Comment by RealRaven2000** [comment]
> Where do we find version 6.5.1? https://addons.thunderbird.net/en-US/thunderbird/addon/filtaquilla/versions/ only shows 4.0.

this is the quickFilters Github. The latest version version of filtaquilla is 4.1 - I just released that and it is currently awaiting review on ATN. If you like a prerelease check here:

https://github.com/RealRaven2000/FiltaQuilla/issues/262#issuecomment-2188490239

**Comment by BillH99999** [comment]
I'm confused.    Above you said "Compatibility with ESR 128 Implemented v**6.5.1** - published 16/07/2024".

**Comment by RealRaven2000** [comment]
> I'm confused. Above you said "Compatibility with ESR 128 Implemented v**6.5.1** - published 16/07/2024".

yes, that was for the Add-on quickFilters - and you are currently looking at the Github for it.

> Where do we find version 6.5.1? https://addons.thunderbird.net/en-US/thunderbird/addon/filtaquilla/versions/ only shows 4.0.

In that question you asked about FiltaQuilla (look at the path of the link), which is a different Add-on and that has 4.1 as latest version (awaiting review by ATN). I took over development of FiltaQuilla for Kent James, who retired from Add-ons due to health reasons a couple of years back. I thought it was a good fit as it has to do with filters just as quickFilters. But it is a different, separate Add-on.

**Comment by BillH99999** [comment]
OK... I see my confusion.  Sorry about that.  I thought I was on the FiltaQuilla issues.  Thanks for the clarification.

**Comment by RealRaven2000** [comment]
Related problems that affect Thunderbird 128:

See issue #258
